### PR TITLE
Docs: Added reference to new timeout parameter pxf.connection.upload-timeout

### PR DESCRIPTION
**Pull request by mperezfuster**
Documenting PR https://github.com/greenplum-db/pxf/pull/644
Let me know if reference to it should be added to some other docs section.
Preview: https://mireia-pxf-timeout.sc2-04-pcf1-apps.oc.vmware.com/pxf/6-1/using/config_files.html

### CODE REVIEW

File: docs/content/config_files.html.md.erb [hunk 1]
```diff
@@ -15,7 +15,8 @@ The `pxf-application.properties` file exposes these PXF Service application conf
 
 | Parameter  | Description | Default Value |
 |-----------|---------------| ------------|
-| pxf.connection.timeout | The Tomcat server connection timeout. (-1 for infinite timeout.) | 5m (5 minutes) |
+| pxf.connection.timeout | The Tomcat server connection timeout for read and write operations (-1 for infinite timeout). | 5m (5 minutes) |
+| pxf.connection.upload-timeout | The Tomcat server connection timeout for write operations only (-1 for infinite timeout). | 5m (5 minutes) |
```
Comment (lisakowen) [inline docs/content/config_files.html.md.erb:19]: we might need an associated advanced configuration topic, let me think about this.

i *think* the default value is the value of pxf.connection.timeout.

**Review by lisakowen — COMMENTED** [review]
how about adding a link to the associated feature PR (644) in the description?  that will link these docs to that feature.  (i forget to do this sometimes, but i think it is good practice.)

File: docs/content/config_files.html.md.erb [hunk 2]
```diff
@@ -15,7 +15,8 @@ The `pxf-application.properties` file exposes these PXF Service application conf
 
 | Parameter  | Description | Default Value |
 |-----------|---------------| ------------|
-| pxf.connection.timeout | The Tomcat server connection timeout. (-1 for infinite timeout.) | 5m (5 minutes) |
+| pxf.connection.timeout | The Tomcat server connection timeout for read and write operations (-1 for infinite timeout). | 5m (5 minutes) |
```
Comment (denalex) [inline docs/content/config_files.html.md.erb:18]:
```suggestion
| pxf.connection.timeout | The Tomcat server connection timeout for read operations (-1 for infinite timeout). | 5m (5 minutes) |
```

File: docs/content/config_files.html.md.erb (hunk 1 above)
Comment (denalex) [inline docs/content/config_files.html.md.erb:19]:
```suggestion
| pxf.connection.upload-timeout | The Tomcat server connection timeout for write operations (-1 for infinite timeout). | 5m (5 minutes) |
```

**Review by denalex — COMMENTED** [review]
the code logic was updated, timeouts are now independent of each other

**Review by ashuka24 — APPROVED** [review]

File: docs/content/config_files.html.md.erb (hunk 2 above)
Comment (denalex) [reply docs/content/config_files.html.md.erb:18]: why was it resolved, but the proposed change has not been made ? This property only manages the read timeout now.

File: docs/content/config_files.html.md.erb (hunk 1 above)
Comment (denalex) [reply docs/content/config_files.html.md.erb:19]: not anymore, we changes the properties to be independent, the default is 5m

File: docs/content/config_files.html.md.erb (hunk 2 above)
Comment (mperezfuster) [reply docs/content/config_files.html.md.erb]: sorry I forgot to push it!

**Review by lisakowen — APPROVED** [review]

**Pull request merged**